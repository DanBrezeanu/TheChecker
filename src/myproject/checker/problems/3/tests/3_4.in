74
3640 4671 7413 5127 8359 2785 346 855 5952 9624 1186 9780 4461 4446 2014 2468 8333 1890 9854 6567 2332 3973 315 6258 2975 8600 3208 7714 6457 7360 4452 4521 3567 8007 7070 5075 7040 3317 9803 6216 2087 9671 2444 6182 4034 7667 3026 7604 4494 5588 378 3824 4203 3676 5739 9252 9427 344 1556 7876 5240 889 257 8010 9390 9728 5306 2254 2262 8059 276 8231 7536 5560 
