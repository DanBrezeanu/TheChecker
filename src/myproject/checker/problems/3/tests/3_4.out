276 344 346 378 1186 1556 1890 2014 2254 2262 2332 2444 2468 3026 3208 3640 3676 3824 4034 4446 4452 4494 5240 5306 5560 5588 5952 6182 6216 6258 7040 7070 7360 7536 7604 7714 7876 8010 8600 9252 9390 9624 9728 9780 9854 
9803 9671 9427 8359 8333 8231 8059 8007 7667 7413 6567 6457 5739 5127 5075 4671 4521 4461 4203 3973 3567 3317 2975 2785 2087 889 855 315 257 
